.github/workflows/deploy.yml

```diff
@@ -0,0 +1,122 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
:question: I guess this will not be used then on stable?

Reply:
Yeah, it'll just not run and do no harm.